internal/fositestorage/accesstoken/accesstoken.go

```diff
@@ -32,7 +32,8 @@ const (
 	// Version 3 is when we added the Username field to the psession.CustomSessionData.
 	// Version 4 is when fosite added json tags to their openid.DefaultSession struct.
 	// Version 5 is when we added the UpstreamUsername and UpstreamGroups fields to psession.CustomSessionData.
-	accessTokenStorageVersion = "5"
+	// Version 6 is when we upgraded fosite in Dec 2023.
+	accessTokenStorageVersion = "6"
```

Comment:
Curious if the "rules" for changing this value are documented anywhere.  
The only place I can think of currently is the release checklist in our CI.  🤔

Reply:
As maintainers, we should bump it whenever the format of the content of those storage secrets is going to change, e.g. due to a change in fosite or in our own code. We don’t want new code to try to read an old secret from an older version which put different content into the secret.

Reply:
👍